### ISSUE TITLE
model_version not set in inference response unless it is set in request

### ISSUE THREAD

**Issue by jeffgriffith** [NVIDIA/triton-inference-server#1806]
Triton Version: 20.03.1-py3
API: gRPC
api-version: 2

It appears that the model_version does not appear in the inference response unless it is explicitly set in the request. Here is our response when we DO NOT explicitly set the inference version in the request:

```
model_name: "mnist"
outputs {
  name: "probabilities"
  datatype: "FP32"
  shape: 1
  shape: 10
  contents {
    raw_contents: "\325\261H7\224\331T;W\024g7\177\256\1778n\204~?u\20588\301\022+9.g\3178\217\265\023:\241\035\320:"
  }
}
```

And here is the inference response when we DO specify the version in the request:

```
model_name: "mnist"
model_version: "1"   <========
outputs {
  name: "probabilities"
  datatype: "FP32"
  shape: 1
  shape: 10
  contents {
    raw_contents: "\325\261H7\224\331T;W\024g7\177\256\1778n\204~?u\20588\301\022+9.g\3178\217\265\023:\241\035\320:"
  }
}
```

It's mainly when no version is specified in the request that we need to know the version actually used for analysis on the client side.

**Comment by deadeyegoodwin** [comment]
Please try with 20.06 version of Triton.

**Comment by jeffgriffith** [comment]
Verified fixed in 20.06.
